.travis.yml

```diff
@@ -42,6 +42,7 @@ matrix:
         # Sphinx is pinned so that new versions that introduce new warnings won't suddenly cause build failures.
         # (Updating the version is fine as long as no warnings are raised by doing so.)
         - python -m pip install sphinx~=1.6.1
+        - python -m pip install blurb
```

Comment:
I pushed this version.  I don't remember what I changed.  I didn't keep the "dev" version on purpose, but then again it is still a little bit under development.